kubernetes/machine-deployment.yaml

```diff
@@ -31,4 +31,10 @@ spec:
       class:
         kind: MachineClass # Machine class template used to create machine, could be AWS/GCP/Azure/Other-cloud-providers
         name: test-mc # Name of the machine class
-        namespace: default
\ No newline at end of file
+        namespace: default
+      # nodeTemplate: 
+      #   metadata:
+      #     annotations: # Optional: allows placing machine on a dedicated host in an availability zone,assuming dedicated host has auto-placement enabled 
+      #       machine.sapcloud.io/awsPlacementAffinity: host #Optional: can't take host unless Tenancy is also host
+      #       machine.sapcloud.io/awsPlacementAvailabilityZone: eu-west-1a #Optional
+      #       machine.sapcloud.io/awsPlacementTenancy: host #Optional
```

Comment:
This makes the `end-user` think that the `nodeTemplate` is a subsection of `class` but when in reaility it's a subsection of `spec` as seen from [this](https://github.com/gardener/machine-controller-manager/blob/master/pkg/apis/machine/v1alpha1/machine_types.go#L63). Can we keep the comments before the alignment? Refer https://github.com/gardener/gardener/blob/master/example/90-shoot.yaml#L49-L67. 

It would be great if you can move the other optional fields similarily [here](https://github.com/gardener/machine-controller-manager-provider-aws/blob/f81603460c30057c71426cab9f53ad1080512db6/kubernetes/machine-deployment.yaml#L13-L15)